src/components/ConfirmationModal.js

```diff
@@ -0,0 +1,51 @@
+import Modal from "./Modal";
+
+class ConfirmationModal extends Modal {
+  constructor(modalSelector) {
+    super({ modalSelector });
+    this._deleteConfirmation = this._modalElement.querySelector(
+      "#delete-confirmation-modal"
+    );
+  }
+
+  open(id) {
+    super.open();
+  }
+
+  close() {
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.